src/SymbolServer.jl

```diff
@@ -42,20 +85,29 @@ function getstore(ssi::SymbolServerInstance, environment_path::AbstractString, p
                     if manifest !== nothing
                         @debug "Downloading cache files for manifest at $(manifest_filename)."
                         to_download = collect(validate_disc_store(ssi.store_path, manifest))
-                        batches = Iterators.partition(to_download, max(1, floor(Int, length(to_download)÷50)))
-                        for (i, batch) in enumerate(batches)
-                            percentage = round(Int, 100*(i - 1)/length(batches))
-                            progress_callback !== nothing && progress_callback("Downloading caches...", percentage)
-                            @sync for pkg in batch
+                        try
+                            remove_non_general_pkgs!(to_download)
+                        catch err
+                            @error """
+                            Symbol cache downloading: Failed to identify which packages to omit based on the General registry.
+                            All packages will be processsed locally""" err
+                            empty!(to_download)
+                        end
+                        if !isempty(to_download)
+                            n_done = 0
+                            @sync for pkg in to_download
```

Comment:
Should the number of concurrent download tasks be limited? Perhaps doesn't matter.

Reply:
Yea maybe not needed.

Reply:
Adding a limit makes sense to me; I think you can get stuff like requests timing out if there’s tons and the cpu can’t service them all

Reply:
Ok. Updated to batches of 100